backend/convert_to_protobuf.go

```diff
@@ -209,24 +209,43 @@ func (t ConvertToProtobuf) RunStreamRequest(req *RunStreamRequest) *pluginv2.Run
 func (t ConvertToProtobuf) StreamPacket(p *StreamPacket) *pluginv2.StreamPacket {
 	protoReq := &pluginv2.StreamPacket{
 		Type:    pluginv2.StreamPacket_StreamPacketType(p.Type),
-		Header:  p.Header,
 		Payload: p.Payload,
 	}
 	return protoReq
 }
 
-// SubscribeToStreamRequest ...
-func (t ConvertToProtobuf) SubscribeToStreamRequest(req *SubscribeToStreamRequest) *pluginv2.SubscribeToStreamRequest {
-	return &pluginv2.SubscribeToStreamRequest{
+// SubscribeStreamRequest ...
+func (t ConvertToProtobuf) SubscribeStreamRequest(req *SubscribeStreamRequest) *pluginv2.SubscribeStreamRequest {
+	return &pluginv2.SubscribeStreamRequest{
 		PluginContext: t.PluginContext(req.PluginContext),
 		Path:          req.Path,
 	}
 }
 
-// SubscribeToStreamResponse ...
-func (t ConvertToProtobuf) SubscribeToStreamResponse(req *SubscribeToStreamResponse) *pluginv2.SubscribeToStreamResponse {
-	return &pluginv2.SubscribeToStreamResponse{
-		Ok:      req.OK,
-		Message: req.Message,
+// SubscribeStreamResponse ...
+func (t ConvertToProtobuf) SubscribeStreamResponse(req *SubscribeStreamResponse) *pluginv2.SubscribeStreamResponse {
+	return &pluginv2.SubscribeStreamResponse{
+		Ok:           req.OK,
+		ErrorMessage: req.ErrorMessage,
+		Schema:       req.Schema,
```

Comment:
In this context, is it really "Schema", or just "first message/packet" 

The fact it is a schema is only true for data streams

Reply:
Changed this to Data which becomes subscribe result data 1-1.